### PR TITLE
New themed modal for option selections

### PR DESCRIPTION
**Pull request by amanalili**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

### CODE REVIEW

File: src/components/common/WalletListMenu.js [hunk 1]
```diff
@@ -1,37 +1,35 @@
 // @flow
```
Comment (amanalili) [inline src/components/common/WalletListMenu.js:1]: Implement all logic to the WalletListModalMenu itself

File: src/components/common/WalletListRow.js [hunk 2]
```diff
@@ -180,14 +192,11 @@ class WalletListRowComponent extends React.Component<Props> {
...
-                <WalletListMenu
-                  currencyCode={guiWallet.currencyCode}
-                  customStyles={customWalletListOptionsStyles}
-                  executeWalletRowOption={this.props.executeWalletRowOption}
-                  walletId={id}
-                />
-              </View>
+              <TouchableWithoutFeedback onPress={this.openWalletListMenuModal}>
+                <View style={styles.rowOptionsWrap}>
+                  <Text style={styles.rowOptionsIcon}>&#8942;</Text>
```
Comment (amanalili) [inline src/components/common/WalletListRow.js:197]: make a const on the top for the option dot

File: src/components/common/WalletListRow.js [hunk 3]
```diff
@@ -237,20 +245,6 @@ class WalletListRowComponent extends React.Component<Props> {
...
-    top: 6
-  },
-  menuIconWrap: {
-    width: scale(46),
-    height: '100%',
-    alignItems: 'center',
-    justifyContent: 'flex-start'
-  }
-})
 const rowCurrencyOverlaySize = scale(23.3)
```
Comment (amanalili) [inline src/components/common/WalletListRow.js:248]: @paullinator should rowCurrencyOverlaySize be inside the stylesheet?

File: src/components/modals/WalletListMenuModal.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+import React, { PureComponent } from 'react'
+import { Image, Text, TouchableOpacity, View } from 'react-native'
+import slowlog from 'react-native-slowlog'
+import AntDesignIcon from 'react-native-vector-icons/AntDesign'
+import { connect } from 'react-redux'
+import { sprintf } from 'sprintf-js'
+
+import { type WalletListMenuKey, walletListMenuAction } from '../../actions/WalletListMenuActions.js'
+import { WALLET_LIST_MENU } from '../../constants/WalletAndCurrencyConstants.js'
```
Comment (amanalili) [inline src/components/modals/WalletListMenuModal.js:11]: Put option unicode constant here

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Fix Art's comments

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**